DXMainClient/DXGUI/Multiplayer/CnCNet/CnCNetLobby.cs

```diff
@@ -690,9 +690,9 @@ private bool JoinGameByIndex(int gameIndex, string password)
         /// <summary>
         /// Attempt to join a game
```

Comment:
Might as well also add a dot here while we're at it.

Reply:
Done.